Gems/Atom/RPI/Code/Tests/ShaderResourceGroup/ShaderResourceGroupConstantBufferTests.cpp

```diff
@@ -389,7 +389,7 @@ namespace UnitTest
             // MyFloat2
             EXPECT_FALSE(m_srg->SetConstantArray<float>(RHI::ShaderInputConstantIndex(13), AZStd::array<float, 3>({ 0.1f, 0.2f, 0.3f })));
 
-            AZ_TEST_STOP_ASSERTTEST(1);
+            AZ_TEST_STOP_ASSERTTEST(2);
```

Comment:
What was this changed?

Reply:
Since it is set now in the (new) multi device and the (preexisting) single device constants data, you get two asserts now.